stores/dropsMinted.ts

```diff
@@ -0,0 +1,29 @@
+import { defineStore } from 'pinia'
+import type { DropItem } from '@/params/types'
+
+// would be nice if we could receive this from the backend
+// https://github.com/kodadot/private-workers/issues/214
+export const useMintedDropsStore = defineStore('mintedDrops', {
+  state: () => ({
+    mintedDrops: [] as DropItem[],
+  }),
+  getters: {
+    sortedMintedDrops(): DropItem[] {
+      return [...this.mintedDrops].sort((a, b) => {
+        // Sort from bigger to lower number
+        return Number(b.collection) - Number(a.collection)
+      })
+    },
```

Comment:
For the drops on base, the id of collection is not the number but the address.

<img width="1174" alt="image" src="https://github.com/user-attachments/assets/3ec960e3-91a0-4c18-a339-463deb5adca0">

Reply:
> `For the drops on base, the id of collection is not the number but the address.`

Well in the end the address is just 20-byte integer ^-^ 

```js
> a = '0x8aedf51f1643296b83dc280680bad43237dfb52a'
'0x8aedf51f1643296b83dc280680bad43237dfb52a'
> b = '0x9789b13d485b124255f9dbf42aa7091aa22e87f8'
'0x9789b13d485b124255f9dbf42aa7091aa22e87f8'
> Number(a) - Number(b)
// -7.198089214047003e+46
```

Reply:
That's true. However, I guess it is supposed to sort the collections by the collection creation time. In other words, sorting by collection ID can not achieve this goal for base drops.

Reply:
updated, let's show it without no sorted method for evm base. since we doesnt have `created_at` data